api/api.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/cenkalti/backoff/v4"
 
-	"github.com/coreos/go-oidc"
+	"github.com/ourrootsorg/go-oidc"
```

Comment:
The plan is to switch that back to the `coreos` version once they accept your PR, correct?

Reply:
That's right.